drivers/sensor/mc3419/mc3419.c

```diff
@@ -298,9 +304,14 @@ static const struct sensor_driver_api mc3419_api = {
 #endif
 
 #define MC3419_DEFINE(idx)						\
+	BUILD_ASSERT(((DT_INST_PROP(idx, decimation_rate) >= 0) &&	\
```

Comment:
I do not think it is possible to trigger this assert due to the enum values specified.

Reply:
build assert is for `decimation-rate`, while `lpf-fc-sel` is enum